examples/koa-http-server-ts/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "koa-http-server-ts",
+  "version": "1.0.0",
+  "description": "async-store example for koajs using typescript",
+  "main": "index.js",
+  "author": "Saroj Rana <rana.saroj95@gmail.com>",
```

```suggestion
  "author": "Swechhya Bista",
  "contributors": ["Saroj Rana <rana.saroj95@gmail.com>"]
```

Can you add your name in contributors list instead of removing.

---

sure, thanks for suggestion

---

Thanks for pointing out @cham11ng :+1:. It's opensource so we should always acknowledge and mention the original author.